cmd/ipfs-cluster-service/main.go

```diff
@@ -239,6 +245,10 @@ remove the %s file first and clean any Raft state.
 					Name:  "custom-secret, s",
 					Usage: "prompt for the cluster secret",
 				},
+				cli.StringFlag{
```

Comment:
There is a `SliceString` type for this. Saves you having to split things

Reply:
Yes, but it is not a user friendly option.

You have to use it like `ipfs-cluster init --peers <multiaddress1> --peers <multiaddress2>`
If you try `ipfs-cluster init --peers <multiaddress1>,<multiaddress2>`, it would assume `<multiaddress1>,<multiaddress2>` to be the first entry in the string slice

Reply:
I had completely missed that. Can you open an issue about this in the places that we use StringSlice?

Reply:
Sure